l2-frontend/src/fields/ConfigureAnesthesiaField.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <div style="margin-top: 10px" v-model="tb_data" :readonly="disabled">
+    <table class="table table-bordered">
+      <thead>
+      <tr>
+        <th class="first-column">Наименование</th>
+        <th class="second-column">Категория</th>
+        <th class="third-column">Показать по умолчанию</th>
+        <th></th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr class="tr-body" v-for="(val, index) in tb_data">
+        <td class="first-column"><input class="no-outline" type="text" style="width:100%" v-model="val['title']"
```

Comment:
Можно и чуть проще написать:
```suggestion
        <td class="first-column"><input class="no-outline" type="text" style="width:100%" v-model="val.title"
```

Reply:
Как и везде в этом файле

Reply:
А ещё не нужен тут класс .first-column, потому что есть :nth-child или :first-child и :last-child

Reply:
И очень странный style тут